### PR TITLE
Change the default ampir model

### PR DESCRIPTION
**Pull request by Darcy220606**
## PR checklist : this PR addresses issue #80 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -35,7 +35,7 @@ params {
     amp_skip_macrel                 = false
 
     amp_skip_ampir                  = false
-    amp_ampir_model                 = 'precursor'
+    amp_ampir_model                 = 'mature'
```
Comment (louperelo) [inline nextflow.config:38]: Just out of curiosity: Why is the 'mature' model better as default?

**Review by louperelo — APPROVED** [review]
Looks good to me.

File: nextflow.config (hunk 1 above)
Comment (Darcy220606) [reply nextflow.config:38]: In the `meta` mode, its more likely that you would detect the `mature` peptides, at least in our case. We already have `meta` invoked in the prodigal/prokka module n the pipeline, so i thought that would make more sense in this case.

Comment (jfy133) [reply nextflow.config:38]: After looking at the documentation we think actualy that `precusor` may still be better as a default value. We will revert this but document in a summary fashion which to use when.

**Review by jfy133 — APPROVED** [review]
Same again here @Darcy220606

**Pull request merged**